src/main/java/UserInputException.java

```diff
@@ -0,0 +1,73 @@
+import java.util.Arrays;
+import java.util.List;
+class UserInputException extends Exception {
+    static void checkUserInput(String input, int currentSize) throws DukeException {
+        String[] inputSplitArray = input.split(" ");
```

Comment:
Perhaps a more intuitive variable name order here like splitInputArray?
```suggestion
         inputSplitArray
```